core/templates/dev/head/components/forms/schema-based-editors/schema-based-html-editor.directive.html

```diff
@@ -1,11 +1,11 @@
 <!-- label-for-focus-target feature is removed since CKEditor has a
 focusManager to manage the focus activity automatically. -->
-<ck-editor-4-rte ng-if="!$ctrl.isDisabled() && $ctrl.isCurrentRTECKEditor_4()" ng-model="$ctrl.localValue"
-                 ui-config="$ctrl.uiConfig()">
-</ck-editor-4-rte>
-<ck-editor-5-rte ng-if="!$ctrl.isDisabled() && !$ctrl.isCurrentRTECKEditor_4()" ng-model="$ctrl.localValue"
+<ck-editor-5-rte ng-if="!$ctrl.isDisabled() && !$ctrl.isCurrentRteCKEditor4" ng-model="$ctrl.localValue"
```

Comment:
Here and elsewhere -- I think I talked before about having functions start with a verb, and non-functions not start with a verb.

So, either $ctrl.isCurrentRteCKEditor4() (if it's a function), or $ctrl.currentRteIsCKEditor4 (if it's a boolean). But not $ctrl.isCurrentRteCKEditor4.

Please fix, here and in a bunch of places elsewhere.

Reply:
it was not a function so renamed it to `currentRteIsCKEditor4`